api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -275,12 +277,36 @@ public void alertUsersUnusedDiskWarningThreshold(
       int daysUnused,
       @Nullable Double workspaceInitialCreditsRemaining)
       throws MessagingException {
-    log.info(
-        Joiner.on(",")
-            .join(
-                new Object[] {
-                  "notifying!", users, diskWorkspace, daysUnused, workspaceInitialCreditsRemaining
-                }));
+    final String htmlMessage =
+        buildHtml(
+            UNUSED_DISK_EMAIL,
+            ImmutableMap.<EmailSubstitutionField, String>builder()
+                .put(EmailSubstitutionField.HEADER_IMG, getAllOfUsLogo())
+                .put(
+                    EmailSubstitutionField.DISK_SIZE,
+                    NumberFormat.getNumberInstance(Locale.US).format(disk.getSize()) + " GB")
+                .put(EmailSubstitutionField.WORKSPACE_NAME, diskWorkspace.getName())
+                .put(EmailSubstitutionField.DISK_UNUSED_DAYS, Integer.toString(daysUnused))
+                .put(
+                    EmailSubstitutionField.DISK_COST_PER_MONTH,
+                    String.format("$%.2f", PersistentDisks.costPerMonth(disk)))
+                .put(
+                    EmailSubstitutionField.DISK_CREATION_DATE,
+                    formatDateCentralTime(Instant.parse(disk.getAuditInfo().getCreatedDate())))
+                .put(EmailSubstitutionField.DISK_CREATOR_USERNAME, disk.getAuditInfo().getCreator())
+                .put(
+                    EmailSubstitutionField.BILLING_ACCOUNT_DETAILS,
+                    buildBillingAccountDescription(diskWorkspace, workspaceInitialCreditsRemaining))
+                .put(EmailSubstitutionField.WORKSPACE_URL, buildWorkspaceUrl(diskWorkspace))
```

Comment:
When user log in, the landing page is always the Home page.

Reply:
Yep. Maybe someday we will fix this. Still, it is worth linking to the correct page. If you open this link in a session where you are already logged in, for example, it will work.